config/validate/report/report.go

```diff
@@ -29,7 +29,7 @@ func (into *Report) Merge(from Report) {
 	into.Entries = append(into.Entries, from.Entries...)
 }
 
```

Comment:
Assuming there's no reason not to use .Interface() above, we can keep EntryKind private yes? I'm not opposed to making it public, but I don't see a need.